src/routes/CreateQuiz/index.tsx

```diff
@@ -32,27 +26,9 @@ export default function ConfigureQuiz() {
       <Helmet>
         <title>Create Quiz</title>
       </Helmet>
-      <form onSubmit={handleSubmit(handleAddQuizName)} style={{ paddingTop: '150px' }}>
-        <Grid align="center" pt="xl" mt="xl">
-          <Grid.Col span={4} offset={4} pt="xl" mt="xl">
-            <FormInput
-              name="name"
-              id="name"
-              register={register}
-              rules={{ required: 'Please enter quiz name' }}
-              errorMessage={errors.name?.message || ''}
-              type="text"
-              placeholder="Please enter quiz name"
-              variant="filled"
-              radius="md"
-              size="md"
-            />
-            <Button mt="xl" radius="sm" size="md" fullWidth type="submit" variant="filled">
-              Create Quiz
-            </Button>
-          </Grid.Col>
-        </Grid>
-      </form>
+      <div style={{ paddingTop: '150px' }}>
```

Comment:
Can we add class instead of inline styling?
Also this additional div seems unnecessary. Can we apply this padding inside the AddOrUpdateQuizName component itself?